<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<class>Prescriptionview</class>
<widget class="QWidget" name="Prescriptionview">
 <property name="geometry">
  <rect>
   <x>0</x>
   <y>0</y>
   <width>1200</width>
   <height>650</height>
  </rect>
 </property>
 <property name="windowTitle">
  <string>处方管理</string>
 </property>
 <layout class="QVBoxLayout" name="verticalLayout">
  <item>
   <layout class="QHBoxLayout" name="filterLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>日期：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateFilter">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>状态：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cmbStatus">
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>搜索：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtSearch">
      <property name="placeholderText">
       <string>处方号、患者姓名或医生姓名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSearch">
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>医生：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cmbDoctor">
     </widget>
    </item>
   </layout>
  </item>
  <item>
   <layout class="QHBoxLayout" name="buttonLayout">
    <item>
     <widget class="QPushButton" name="btnAdd">
      <property name="text">
       <string>开具处方</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnView">
      <property name="text">
       <string>查看详情</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEdit">
      <property name="text">
       <string>编辑</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDispense">
      <property name="text">
       <string>发药</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPayment">
      <property name="text">
       <string>收费</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPrint">
      <property name="text">
       <string>打印处方</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnExport">
      <property name="text">
       <string>导出</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAudit">
      <property name="text">
       <string>审核</string>
      </property>
     </widget>
    </item>
   </layout>
  </item>
  <item>
   <widget class="QTableView" name="tableView">
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
  </item>
  <item>
   <layout class="QHBoxLayout" name="statsLayout">
    <item>
     <widget class="QLabel" name="lblTodayPrescription">
      <property name="text">
       <string>今日处方：0张</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblTodayAmount">
      <property name="text">
       <string>今日金额：0.00元</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblPendingDispense">
      <property name="text">
       <string>待发药：0张</string>
      </property>
      <property name="styleSheet">
       <string notr="true">color: orange; font-weight: bold;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblUnpaid">
      <property name="text">
       <string>待收费：0张</string>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red; font-weight: bold;</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="lblMonthAmount">
      <property name="text">
       <string>本月金额：0.00元</string>
      </property>
     </widget>
    </item>
   </layout>
  </item>
 </layout>
</widget>
<resources/>
<connections/>
</ui>